### PR TITLE
improve naming of hard swish functions

### PR DESCRIPTION
**Pull request by brettkoonce**
per discussion in #743, this helps make clearer what is going on.

### CODE REVIEW

**Comment by sgugger** [comment]
Agreed that those names are better, thanks @t-ae for the suggestion.

File: Sources/TensorFlow/Operators/Math.swift [hunk 1]
```diff
@@ -1435,7 +1435,7 @@ public func hsigmoid<T: TensorFlowFloatingPoint>(_ x: Tensor<T>) -> Tensor<T> {
 /// https://arxiv.org/abs/1905.02244
 @inlinable
 @differentiable
-public func hswish<T: TensorFlowFloatingPoint>(_ x: Tensor<T>) -> Tensor<T> {
+public func hardSwish<T: TensorFlowFloatingPoint>(_ x: Tensor<T>) -> Tensor<T> {
   x * hsigmoid(x)
```
Comment (sgugger) [inline Sources/TensorFlow/Operators/Math.swift]:
```suggestion
  x * hardSigmoid
```

Comment (brettkoonce) [reply Sources/TensorFlow/Operators/Math.swift]: doh 😓

**Comment by brettkoonce** [comment]
Step 20/37 : RUN /swift-tensorflow-toolchain/usr/bin/swift test
 ---> Running in cbbcabccc3c9
[91mSegmentation fault (core dumped)
[0mThe command '/bin/sh -c /swift-tensorflow-toolchain/usr/bin/swift test' returned a non-zero code: 139

tests haven't been running for me for a week or two, dying in the lazytensor area.  if i comment them out the above passes, fwiw.  thanks as ever!

**Comment by sgugger** [comment]
Finally, the tests pass. Thanks for the PR!

**Pull request merged**